core/src/components/badge/badge.tsx

```diff
@@ -41,7 +41,7 @@ export class TdsBadge {
         ? console.warn(
             'The provided value is either empty or string, please provide value as number.',
           )
```

Comment:
Do we really need a turnary here? Couldn't we just do:

if(this.value !== '' && this.size !== 'sm'){
console.warn('The provided value is either empty or string, please provide value as number.')
}

Reply:
Yeah, maybe, we can try with that instead.